src/simulation/dynamics/Thrusters/thrusterDynamicEffector/_UnitTest/test_thruster_integrated_sim.py

```diff
@@ -152,6 +154,11 @@ def thrusterIntegratedTest(show_plots):
             testFailCount += 1
             testMessages.append("FAILED: Thruster Integrated Test failed attitude unit test")
 
+    accuracy = 1e-9
+    if not unitTestSupport.isDoubleEqual(thrustersDynamicEffector.fuelMass, fuelLog.fuelMass[-1], accuracy):
```

Comment:
:small_red_triangle_down: We consider `unitTestSupport` deprecated. Use numpy.testing asserts instead.

If possible we should convert this test numpy.testing asserts and delete all the `testFailCount` pieces.

Reply:
This has been done for this integrated test.